Readme.md

```diff
@@ -50,26 +50,34 @@ Explorer for Endevor is included with Eclipse Che version 7.6.0 and above. Check
 
 Create a Host in Explorer for Endevor to enable it to access the repository you want to explore.
 
-### **Create a Host**
+### **Create a Profile**
 
-Create a Host in Explorer for Endevor to specify the data source you want to work with. This allows you to view and explore the selected Endevor repositories.
+Create a Profile in Explorer for Endevor to specify the data source that you want to work with. This allows you to view and explore the selected Endevor repositories.
 
+**Follow these steps:**
+
+1. Click on the + icon.
+2. Enter a name for your Profile.
+3. Enter your Endevor URL in the format `https://host:port`.
+4. (Optional) To add your mainframe credentials to your profile, enter your username and password. Adding your credentials to your profile lets you access different data sources without entering your credentials.
+5. Set profile settings to True or False to indicate that your profile should Reject or Accept connections with self-signed certificates.  
+Your Profile is now available in the panel on the left.
 
-![](https://github.com/eclipse/che-che4z-explorer-for-endevor/blob/v0.9.1/create_host.gif)
+6. Click on the profile in the panel.  
+The profile automatically populates in the terminal panel.
 
-**Procedure:**
+7. To add a new data source, click + next to the panel and select the required additional data source.  
+Your data source appears in the panel below the profile entry.  
+This step can be repeated as many times as you need to add multiple data sources.
 
-1. Open the Explorer for Endevor extension.
-2. Click the plus icon or *New connection*.
-   A prompt opens.
-3. Enter an Endevor repository URL in the format 'http(s)://url:port', for example 'http(s)://myhost01:9090' then press 'Enter' to confirm or 'Escape' to cancel.
-4. Select a CA Endevor SCM configuration from the drop-down list.
+8. In addition to adding multiple data sources to profiles, you can also add additional spearate profiles. Once created, the profiles are listed in the panel on the left hand side, and can be expanded to show which data sources are attached to them.
```

Comment:
I think this is all a bit obvious. If we must mention that multiple profiles are supported we can do it in a few words.